kibana.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kibana
+  labels:
+    component: kibana
+    role: web
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: kibana
+        tier: frontend
+    spec:
+      containers:
+      - image: kibana:4.5
+        name: kibana
+        env:
+        - name: ELASTICSEARCH_URL
+          value: http://10.0.0.110:9200
```

Comment:
This is the cluster IP of the elastic search service. I did not understand the comment that "this can be any IP address"